.vscode/settings.json

```diff
@@ -52,4 +52,19 @@
     "source.fixAll.eslint": true
   },
   "eslint.validate": ["javascript", "typescript"],
+  // Set the default
```

Comment:
.JSON does not support comments. Consider dropping them?

Reply:
It's only for vscode, which is basically jsonc. Since it's a settings file, I think comments are appropriate.